www/include/eventLogs/template/viewLogEngine.ihtml

```diff
@@ -25,12 +25,16 @@
                 <h4>{t}From{/t}</h4>
                 {$form.StartDate.html}
                 {$form.StartTime.html}
-                {$form.alternativeDateStartDate.html}
+                {if isset($form.alternativeDateStartDate)}
```

Comment:
this one is mandatory to be able to localize the date.
Shouldn't you set it if not set ?